portal.go

```diff
@@ -2186,6 +2245,29 @@ func (portal *Portal) HandleiMessageTapback(msg *imessage.Message) {
 	}
 }
 
+func (portal *Portal) HandleMessageRevoke(msg imessage.Message) bool {
+	dbMessage := portal.bridge.DB.Message.GetByGUID(portal.GUID, msg.GUID, 0)
```

Comment:
This also looks like it won't work for multipart messages

Reply:
hmm, I think the msg has a part property on it in this context, yes? I'll adjust and see if that is better.